annif/openapi/annif.yaml

```diff
@@ -314,6 +460,105 @@ components:
                 type: string
                 example: Vulpes vulpes
       description: A document with attached, known good subjects
+    ReconcileMetadata:
```

Comment:
It could be better to call this type ReconciliationServiceManifest (although it's pretty long...) so that it matches the reconciliation API spec.